src/CodeModel.cpp

```diff
@@ -154,9 +154,8 @@ CompiledContract::CompiledContract(const dev::solidity::CompilerStack& _compiler
 	m_contract.reset(new QContractDefinition(nullptr, &contractDefinition));
 	QQmlEngine::setObjectOwnership(m_contract.get(), QQmlEngine::CppOwnership);
 	m_contract->moveToThread(QApplication::instance()->thread());
-	eth::LinkerObject const& object = _compiler.object(_contractName.toStdString());
-	if (object.linkReferences.empty())
-		m_bytes = object.bytecode; //@todo handle unlinked objects
+	m_linkerObject = _compiler.object(_contractName.toStdString());
+	m_bytes = m_linkerObject.bytecode; // use linkLibraries() to link object
```

Comment:
Is there a good reason to have the duplication between `m_bytes` and `m_linkerObject.bytecode`?